app/models/user.rb

```diff
@@ -120,6 +120,12 @@ def chat_id
     "user_#{id}"
   end
 
+  def is_noticed_in_30_minutes?
```

Comment:
__[RuboCop]__

Naming/PredicateName: Rename `is_noticed_in_30_minutes?` to `noticed_in_30_minutes?`.


<!-- Comment by [Sider](https://sider.review) -->